js/carousel.js

```diff
@@ -418,7 +429,11 @@
      * @param {Number} x
      */
     _wrap(x) {
```

Comment:
causes divide by zero when this.count equals 0

Reply:
same as comment below.